src/plots/polar/constants.js

```diff
@@ -15,15 +15,17 @@ module.exports = {
     axisNames: ['angularaxis', 'radialaxis'],
     axisName2dataArray: {angularaxis: 'theta', radialaxis: 'r'},
 
-    // TODO should radial axis be above frontplot by default?
     layerNames: [
         'draglayer',
         'plotbg',
         'backplot',
-        'grids',
-        'axes',
-        'lines',
-        'frontplot'
+        'angular-grid',
+        'radial-grid',
+        'frontplot',
+        'angular-axis',
+        'radial-axis',
+        'angular-line',
+        'radial-line'
```

Comment:
cc @alexcjohnson 's 

> I also liked the way our old polar plots had the radial axis on top of the data, with an outline around the text in the background color. Perhaps it could be configurable whether this axis is above or below the data?

from https://github.com/plotly/plotly.js/pull/2200#issuecomment-351127938